packages/nextra-theme-blog/src/basic-layout.tsx

```diff
@@ -9,10 +9,7 @@ export const BasicLayout = ({ children }: { children: ReactNode }) => {
   const title = `${opts.title}${config.titleSuffix || ''}`
   const ref = useRef<HTMLHeadingElement>(null)
   return (
-    <article
-      className="nx-container nx-prose-sm dark:nx-prose-dark md:nx-prose"
-      dir="ltr"
-    >
+    <article className="nx-container nx-prose dark:nx-prose-dark" dir="ltr">
```

Comment:
@B2o5T I've removed the `nx-prose-sm` class completely, as it didn't do anything in particular concerning breakpoints. Instead I've added the styles to hide backticks from inline code to the `nx-prose` class. If they're supposed to be shown on bigger breakpoints I can update the class, but tbh I like the code without backticks.

Reply:
> as it didn't do anything in particular concerning breakpoints

it did text small on small breakpoints

Reply:
You're right, had a brainfart and didn't realize `prose-sm` was a tailwind class too. Thanks for adding it.